### ISSUE TITLE
Legacy `printPageNumbers` should be adressed in the EPUB techniques

### ISSUE THREAD

**Issue by gautierchomel** [w3c/publ-a11y#703]
As per [Schema.org Accessibility Properties for Discoverability Vocabulary 4.2.1.5 pageBreakMarkers](https://www.w3.org/community/reports/a11y-discov-vocab/CG-FINAL-vocabulary-20241209/#pageBreakMarkers):

> The value `printPageNumbers` is considered a synonym of `pageBreakMarkers`. Although its use is not formally deprecated, authors are strongly encouraged to use the more general `pageBreakMarkers`. 

The Display Techniques for EPUB Accessibility Metadata 2.0 have instructions to process only `pageBreakMarkers`, but should consider both.
